src/assests/styles/colors.css

```diff
@@ -3,11 +3,13 @@
 :root {
     --primary: #F8F9FA;
     --theme: #212529;
+    --black: black;
```

Comment:
Its always better to use #000 instead of keyword black.

Reply:
ok I will change it